defaults/main.yml

```diff
@@ -9,7 +9,7 @@ iptables_deny_all : yes                 # Deny all except allowed
 iptables_confdir: /etc/iptables
 iptables_rules_path: "{{ iptables_confdir }}/rules.v4"  # Path to rule file
 
-iptables_load_path: /etc/network/if-up.d/iptables_load # Set empty to prevent loading
+iptables_load_path: "{% if  ansible_os_family == 'Debian' %}/etc/network/if-up.d/iptables_load{% elif ansible_os_family == 'RedHat' %}/etc/sysconfig/iptables-config{% endif %}" # Set empty to prevent loading
```

Comment:
Wouldn't it be cleaner to put these in vars/RedHat.yml / Debian.yml? And is the "# Set empty to prevent loading" comment still relevant?

Reply:
Yeah, the comment is still true, although I don't really see a scenario where you would want that.

I'm not a 100% sure, but I think from the `vars` and `defaults` in roles, only the `main.yml` file is loaded automatically, whereas in `group_vars` and `host_vars` everything in the folders is loaded (the [variable precedence](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#understanding-variable-precedence) seems to suggest so, using `*` wildcards in one place and `main.yml` in the other).

So you would have to conditionally load it in the role and then you end up with the problem that I fixed here, that it overrides values you may have set in `host_vars` or `group_vars`

Reply:
Ah, ok! I *think* I understand. Let's keep this as-is if that's the best option.